Rota-Process.md

```diff
@@ -0,0 +1,328 @@
+  [respective repository](https://github.com/kata-containers/), however some are
+  more general (such as `team/developer).
+
+  If you know the GitHub team a particular label relates to, please add a
+  comment including the name of the team using the `@team-name` syntax. All
+  teams are listed [here](https://github.com/orgs/kata-containers/teams). If you
+  do not have access to this list, please [contact the team](README.md#join-us)
+  who will either grant you access or tell you wish team to specify.
+
+- Related project
+
+  If the issue relates to another project, add one of the `related/*`
+  labels.
+
+- Issue is lacking something
+
+  If you think the issue is incomplete somehow, consider adding one of the
+  `needs-*` labels.
+
+##### General process
+
+We use
+[GitHub issue templates](https://github.com/kata-containers/.github/tree/master/.github/ISSUE_TEMPLATE)
+to help guide users to raising the correct type of issue. All these templates
+also automatically add a `needs-review` label to the issues.
+
+The idea of this label is to highlight issues that have *not* yet been
+reviewed by the team.
+
+Therefore, after you have reviewed an issue with this label (which will
+potentially have resulted in you applying additional labels), you should
+*remove* the `needs-review` label to signal that the issue has been reviewed.
+
+### Test stability
+
+The more PRs you look at, the more likely you are to notice patterns or
+anomalies of CI test failures.
+[Raising issues in the `tests` repository](https://github.com/kata-containers/tests/issues/new)
+helps us to identify "flaky" tests and prioritize the problematic bugs that
+make review/merges slower.
+
+Before raising an issue, search all issues (open and closed) looking for the
+key failure message to see if it has been raised before using the page below:
+
+- https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+org%3Akata-containers+
+
+  Enter an error message in the search box and press enter.
+
+If an issue has already been raised, reference it by adding a comment on the
+PR. If not, raise a new issue providing details of the failure and reference
+it by adding a PR comment.
+
+### Weekly Summary
+
+The Team should discuss amongst themselves and nominate someone to write a
+brief summary of what the team accomplished in the week, and post it to
+[the public mailing list](README.md#join-us). This keeps the rest of the
+community informed.
+
+Do not worry if it has been a quiet week. A simple email is still beneficial
+so that the community can see that we are continuing to try to progress PRs.
+
+There is no strict format for the weekly summary but it should mention
+significant PRs that have landed (or been progressed). The summary does not
+need to be highly detailed or particularly long but it does help to include PR
+URLs to allow readers to find out further information.
+
+There is a template if you wish to use it:
```

Comment:
Line 317 suggested rewrite: 
The following link provides a template if you prefer to use one: